section4/exercises/objects.js

```diff
@@ -17,30 +17,62 @@ var foods = {
 };
 
 console.log(foods);
-// Notice that the variable name is foods, plural, because it has the ability to hold info about 
-// potentially many foods. That plural name indicates to another developer that the variable stores 
+// Notice that the variable name is foods, plural, because it has the ability to hold info about
+// potentially many foods. That plural name indicates to another developer that the variable stores
 // either an Array or an Object.
 
 //-------------------
 // Part 1: Zoo Animals
 //-------------------
 
-// YOU DO: Write code below that will declare a variable that stores an Object Literal of animal names and 
+// YOU DO: Write code below that will declare a variable that stores an Object Literal of animal names and
 // the number of that type of animal at the zoo.
-
+var zooAnimals = {
+  zebras: 14,
+  elephants: 23,
+  panthers: 14,
+  gorillas: 4,
+  giraffes: 9,
+}
 
 // Using the zoo that you created above, log the value of the first item in
 // the Object
-
+console.log(zooAnimals.zebras);
```

Comment:
✅

Reply:
Challenge Yourself On Your Own: If you didn't know that the first item was `zebras`, how would you call it?